mindsdb/integrations/handlers/access_handler/tests/test_access_handler.py

```diff
@@ -1,32 +1,436 @@
+import sys
 import unittest
+from unittest.mock import MagicMock, patch
+import pandas as pd
+from mindsdb.integrations.libs.response import HandlerStatusResponse as StatusResponse, RESPONSE_TYPE
+
 from mindsdb.integrations.handlers.access_handler.access_handler import AccessHandler
-from mindsdb.api.executor.data_types.response_type import RESPONSE_TYPE
 
+# Mock pyodbc and sqlalchemy_access before importing AccessHandler to avoid import errors
+sys.modules["pyodbc"] = MagicMock()
+sys.modules["sqlalchemy_access"] = MagicMock()
+sys.modules["sqlalchemy_access.base"] = MagicMock()
+
+
+class BaseAccessHandlerTest(unittest.TestCase):

```

Comment:
We should delete this tests dir and move the tests as other handlers in https://github.com/mindsdb/mindsdb/tree/main/tests/unit/handlers

Reply:
Done